### ISSUE TITLE
Constructing a ride should automatically close it

### ISSUE THREAD

**Issue by devnoname120** [OpenRCT2/OpenRCT2#2915]
This is annoying to click on the construction button, only to be asked to close it first. I think that the ride should automatically be closed, and maybe reopened (if possible) once the user closes the construction tool.

**Comment by Gymnasiast** [comment]
I disagree. This just makes it easier to accidentally throw everyone off your coaster.

**Comment by devnoname120** [comment]
Maybe a double-click on the construction button?

**Comment by Gymnasiast** [comment]
Either that or Ctrl+click (like OpenTTD) would be fine IMO.

**Comment by IntelOrca** [comment]
I often right click a track by accident. Thus the error poping up saying to close the ride is a useful safety net.

**Comment by Patrik356b** [comment]
We could have an option for this hidden in the settings file or something. I agree with both sides but prefer the warning

**Comment by devnoname120** [comment]
@IntelOrca It happens to me too. It might only behave this way with the construction button, since one is unlikely to click on it by mistake.
